README.md

```diff
@@ -16,19 +21,45 @@ This is enabled by using the MFSample Publisher which published IMFSample pointe
 
 To run this node, a camera will be required to be installed and ready to use on your system.
 
+### Simple camera node
 You can begin with the below launch file. It will bring up RViz tool where you can see the image stream from your camera.
 
 ```
 roslaunch msft_camera test/camnode.launch
 ```
 
+### Camera node with RTSP/RTP streaming
+To enable RTSP streaming you will need to create a recursive clone of this repository using the following command:  
+```
+git clone https://github.com/ms-iot/ros_msft_camera --recursive
+```
+
+Then at the top level folder (i.e. parallel to the src folder) use the command  
+
+```
+catkin_make -DENABLE_RTSP=1
+```
+This will build the node with RTP/RTSP streaming capability. 
+To launch the node with streaming enabled, use the following command:
+
+```
+roslaunch msft_camera_rtsp test/camnodeRTSP.launch
+``` 
+After the node has been successfully launched with RTSP enabled, you can view the video on any streaming video player that supports RTSP using the url:  
+> `rtsp://<ip-address-of-machine-running-the-node>:<rtsp_port>`
+
+If using secure RTSP over TLS, use a video player that supports RTSPS with following url:  
```

Comment:
Is there a way to disable the insecure rtsp stream? (I didn't see that explicitly called out)

Reply:
This is controlled by the "rtsps_cert_subject" parameter.
I made the parameter description more detailed to explain this.